setup.py

```diff
@@ -45,6 +45,11 @@
 hyperopt_install_requires = ["hyperopt>=0.2.7"]
 lite_install_requires = ["sigoptlite>=0.1.1"]
 
+if sys.version_info[0] == 3 and sys.version_info[1] < 12:
+  pytest_requires = ["pytest==7.2.1"]
+else:
+  pytest_requires = ["pytest>=7.2.1"]
+
```

Comment:
Newer pytests don't support older pythons?

Reply:
When I ran it I got a whole bunch of errors, but that actually seems to have been another issue- I just tested and it worked fine. Will kill these checks and just unpin pytest in the dev file.